source/vsts.md

```diff
@@ -9,9 +9,13 @@ This extension provides Build and Release tasks to integrate with [Octopus Deplo
 
 You will need a minimum build agent version of `2.115.0` with .NET Core SDK `2.0` or later. When targeting build agents without the SDK, you can use the **.NET Core SDK Installer** task to install it. Generally the Hosted Linux, Mac and Hosted VS2017 agent queues already provide it, however please refer to Microsoft documentation regarding what capabilities are provided by which hosted agent pools.
 
-The Octopus tasks will automatically download, cache, and use the latest version of the Octopus tools, unless they are supplied using the **Octopus tools installer** task, or found using the `PATH` environment variable.
+## Add Octo.exe Command Line tools
 
-If the **Octopus tools installer** task version is set to `embedded`, it will supply an embedded copy of the tools instead of downloading.
+The Octopus tasks require the Octo.exe Command Line tools, which can be supplied any of the following ways:
+
+* Insert the **Octopus tools installer** task into the build pipeline. If you leave the version as `embedded`, a built-in copy of the tools will be supplied to the other tasks. If you set the version to `latest` or a specific number, that version will be downloaded and supplied to the other tasks. If the download fails, the built-in copy of the tools will be used.
```

Comment:
Maybe add an example of a specific number ... `or a specific number like 6.7.0, that version ...`

Reply:
Agreed.